### DISCUSSION TITLE
Popover on animation end call back?

### DISCUSSION THREAD

**Discussion by emadabdulrahim** [radix-ui/primitives#1149] [Help] [answered]
How can I wait for Popover animation to end without having to use setTimeout etc?

**Comment by jjenzz** [comment, marked as answer]
I'll move this to a bug because I fully expected this to work https://codesandbox.io/s/recursing-mountain-oj7hh?file=/src/App.tsx

**Reply by emadabdulrahim** [reply]
Thanks @jjenzz

**Reply by jjenzz** [reply]
created here https://github.com/radix-ui/primitives/issues/1150
